docs/agent-vs-aea.md

```diff
@@ -117,9 +117,7 @@ We run the agent from a different thread so that we can still use the main threa
 We use the input and output text files to send an envelope to our agent and receive a response
 ``` python
         # Create a message inside an envelope and get the stub connection to pass it into the agent
-        message_text = (
-            b"my_agent,other_agent,fetchai/default:0.5.0,\x08\x01*\x07\n\x05hello,"
-        )
+        message_text = b"my_agent,other_agent,fetchai/default:0.5.0,\x12H\x08\x01\x12\x011*A\n4type.googleapis.com/fetch.aea.Default.DefaultMessage\x12\t*\x07\n\x05hello,"
```

Comment:
So our new format is much less efficient...

Reply:
and it doesn't even contain the full dialogue references!

Reply:
@DavidMinarsch UPDATE: please see ca534b0